src/index.js

```diff
@@ -102,6 +103,18 @@ class DataTableExtensions extends Component {
     Utilities.print(table);
   }
 
+  checkHeader() {
+    const { columns } = this.state;
+    if (columns.length !== this.raw.header) {
```

Comment:
what exactly did you want to do here? not an accurate comparison!

Reply:
My bad, I am fixing this condition.